.vuepress/docs-sidebar.js

```diff
@@ -1,124 +1,139 @@
+      ''
+    ]
+  },
+  {
+    title: 'Concepts',
+    collapsable: false,
+    children: [
+      ['concepts/', 'Concepts Overview'],
+      'concepts/things',
+      'concepts/items',
+      'concepts/discovery',
+      'concepts/audio',
+      'concepts/units-of-measurement',  // from v2.3 onwards
+    ]
+  },
+  {
+    title: 'Installation Guide',
+    collapsable: false,
+    children: [
+      'installation/',
+      ['installation/linux', 'Linux'],
+      ['installation/windows', 'Windows'],
+      ['installation/macos', 'macOS'],
+      'installation/openhabian',
+      'installation/rasppi',
+      'installation/pine',
+      'installation/docker',
+      'installation/synology',
+      'installation/qnap',
+      ['installation/security', 'Security']
+    ]
+  },
+  {
+    title: 'New User Tutorial',
+    collapsable: false,
+    children: [
+      ['tutorial/', 'Tutorial Overview'],
+      'tutorial/1sttimesetup',
+      'tutorial/uis',
+      'tutorial/configuration',
+      'tutorial/sitemap',
+      'tutorial/rules',
+      'tutorial/logs',
+      ['tutorial/persistence', 'Persistence']
+    ]
+  },
+  {
+    title: 'Configuration Guide',
+    collapsable: false,
+    children: [
+      ['configuration/', 'Configuration Overview'],
+      'configuration/addons',
+      'configuration/things',
+      'configuration/items',
+      'configuration/sitemaps',
+      'configuration/persistence',
+      'configuration/rules-dsl',
+      'configuration/transformations',
+      'configuration/actions',
+      'configuration/iconsets/classic/',
+      'configuration/jsr223',
+      'configuration/services',
+      'configuration/multimedia',
+      ['configuration/migration/', 'Migration from 1.x']
+    ]
+  },
+  {
+    title: 'Interfaces and Ecosystem',
+    collapsable: false,
+    children: [
+      'configuration/packages',
+      'configuration/editors',
+      'configuration/homebuilder',
+      ['configuration/paperui', 'Paper UI'],
+      ['configuration/ui/habmin/', 'HABmin'],
+      ['configuration/ui/habot/', 'HABot'],
+      'configuration/habpanel',
+      ['configuration/ui/basic/', 'Basic UI'],
+      ['configuration/ui/classic/', 'Classic UI'],
+      'configuration/rules-ng',
+      'configuration/eclipseiotmarket',
+      ['configuration/restdocs', 'REST API'],
+      'apps/android',
+      'apps/ios',
+      'apps/windows',
+      ['ecosystem/alexa/', 'Amazon Alexa'],
+      ['ecosystem/google-assistant/', 'Google Assistant'], // from v2.3 onwards
+      ['../addons/integrations/homekit/', 'Apple HomeKit'],
+      ['ecosystem/ifttt/', 'IFTTT'],
+      ['ecosystem/mycroft/', 'Mycroft.AI'],
+    ]
+  },
+  {
+    title: 'Administration Guide',
+    collapsable: false,
+    children: [
+      'administration/',
+      'administration/console',
+      'administration/runtime',
+      'administration/bundles',
+      'administration/logging',
+      'administration/jsondb',
+    ]
+  },
+  {
+    title: 'Addon Development',
```

Comment:
I'd stick to "Developer Guide" as a heading for this section and remove the (anyhow far too long) "Additional Developer Information" heading.

Reply:
I explained the reason for the split in the PR description. We definitely need two sections, but the headings might need to change. It's the best ones that I came up with at the time of writing.

The idea is that we have one read-through guide. Like the Rust developer handbook if you have ever read that. It completely explains the Rust language in a single book like document.

I like one section to be like that, beginning with basics like git, checkout, setup then explaining all different extension points we have with the general guidelines at the beginning (or the end, not sure yet) and closing with how-to-publish.

The second section is more like an on-demand lookup. Along the lines of "If you want more details about OSGi, we got you covered". All the pieces that you don't need to develop bindings.

I don't think that you can mix up those two, I found that quite confusing in the old structure to be honest.

Reply:
Still, "Addon Development" is not a correct title here as the IDE setup and other development related stuff is likewise valid for core development. Thus I'd keep it as "Developer Guide".

Absolutely ok for me to sort "appendix" stuff to the end. Maybe a separator can be added inbetween. If we want to treat it as a book with multiple chapters, we should rather think about separating it (again) from the user guide (a structure we once had and which imho makes a lot of sense).

Reply:
I'm all for separating it. But as long as we haven't done that we need to work with the limited sidebar 2-level hierarchy to structure topics.

I would clearly not have a dedicated section for "core development" IMO. Not at the moment. Core doesn't know where it wants to head, there is no roadmap, no vision.

If we expose all extension points of Core, people are actually performing core development more or less. The current document already spreads in references to Core classes all the time which helps to make people familiar with the Core structure.

Reply:
I wasn't suggesting a separate "core" section, just making the developer section a general developer section, just as it is now. "Developer Guide" therefore is a perfect match - pretty much the standard label any developer would expect to find.